src/vivarium_csu_ltbi/tools/ltbi_results_postprocessing.py

```diff
@@ -90,6 +144,50 @@ def get_person_time(data):
     all_person_time, ltbi_person_time = split_and_aggregate_person_time(data)    
     return sort_data(all_person_time), sort_data(ltbi_person_time)
 
+def get_risk_specific_population(data, location):
+    pt, ltbi_pt = get_person_time(data)
+    pt['location'] = location
+    pt = aggregate_over_treatment_group(pt)
+    index_cols = ['location', 'year', 'age', 'sex', 'cause', 'risk_group', 'scenario', 'treatment_group']
+    pt = pivot_and_summarize(pt, index_cols)
+    # 2019 baseline mean person_time
+    pt = (pt
+          .reset_index()
+          .query('year == "2019" & scenario == "baseline"')
+          .filter(['location', 'age', 'sex', 'risk_group', 'mean']))
+    
+    pt_cols = ['location', 'age', 'sex']
+    pt_plwhiv = (pt
+                 .loc[pt.risk_group == 'plwhiv']
+                 .drop(columns='risk_group')
+                 .set_index(pt_cols))
+    pt_u5_hhtb = (pt
+                  .loc[pt.risk_group == 'u5_hhtb']
+                  .drop(columns='risk_group')
+                  .set_index(pt_cols))
+    pt_all_population = (pt
+                         .loc[pt.risk_group == 'all_population']
+                         .drop(columns='risk_group')
+                         .set_index(pt_cols))
+    
+    plwhiv_prop = pt_plwhiv / pt_all_population

```

Comment:
For calculating proportion, do you use person-time unit or count unit?

Reply:
I used person time measure from sim output.

Reply:
Thank you for confirming it. For calculating U5 HH AcTB exposure proportion, I used counts of U5 with HH TB exposure/ counts of total U5 children and the counts are for all years (from start year to 2020). @aflaxman Do you think I need to update my script to make it consistent with Yongquan's .

Reply:
I think the person-years measure is a count space metric.